notebook.ipynb

```diff
@@ -1,5 +1,12 @@
 {
```

Comment:
<p>It might also be good to add one additional sentence at the beginning of the description to expand upon the purpose slightly, for people who haven't read the paper yet or have just come accross the notebook. E.g. something like "This notebook explores the process of using a convolutional neural network to attribute forcings to changes in global mean surface temperatures (1900 - 2010), based on observations and climate modelling". </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eds-book-gallery/CI2023-RC-team2/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>I think it is done too.</p>